frontend/packages/noobaa-storage-plugin/src/components/namespace-store/create-ns.tsx

```diff
@@ -0,0 +1,493 @@
+import * as classNames from 'classnames';
```

Comment:
A lot of code duplication from create-bs. Please refactor and import these components.

Reply:
As discussed earlier, I am aware of code duplication as Namespace store and Backing store only differ on Providers(No GCP or PVC). The code needs major refactor in backing store + tests everywhere. As we are short of time and bandwidth, let us focus the reviews for the feature itself. I will take an AI on me for refactoring this later on ;)

Reply:
I do not see a need for refactoring much tbh. Add props to the other components and enhance them to support both the use cases. We still have time to get this feature in. So I would prefer we do it the right way.

Reply:
Ok i have refactored for both.. Everything is working as per my tests. Please review and check